npm/design-system/tsconfig.json

```diff
@@ -5,7 +5,7 @@
     "module": "commonjs" /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */,
     "skipLibCheck": true,
     "lib": [
-      "es2015",
+      "es2016",
```

Comment:
Do we have actual requirements here? Can we just go up to `ESNext`?

Reply:
Array.includes!